app/bundles/LeadBundle/EventListener/DashboardSubscriber.php

```diff
@@ -492,5 +498,77 @@ public function onWidgetDetailGenerate(WidgetDetailEvent $event): void
 
             return;
         }
+
+        if ('number.contacts' == $event->getType()) {
+            $numContacts = $this->leadModel->getNumberContacts();
+            $subtitle    = $this->translator->trans('mautic.widget.number.contacts.description');
+
+            $event->setTemplateData([
+                'value'    => $numContacts,
+                'subtitle' => $subtitle,
+            ]);
+
+            $event->setTemplate('@MauticCore/Helper/single_info.html.twig');
+            $event->stopPropagation();
+
+            return;
+        }
+
+        if ('segment.number.contacts' == $event->getType()) {
+            $params    = $event->getWidget()->getParams();
+            $segmentId = $params['segmentId'] ?? null;
+
+            $segmentRepository = $this->leadListModel->getRepository();
+            $segments          = $segmentRepository->getEntities();
+            $selectedSegment   = null;
+
+            foreach ($segments as $segment) {
+                if ((int) $segment->getId() === (int) $segmentId) {
+                    $selectedSegment = $segment;
+                }
+            }
+
+            if (!empty($segmentId)) {
+                $leadCounts = $this->leadListModel->getSegmentContactCountFromCache([$segmentId]);
+            } else {
+                $leadCounts = [];
+            }
+
+            $count = $leadCounts[$segmentId] ?? 0;
+
+            $subtitle = $this->translator->trans('mautic.widget.segment.number.contacts.description');
+
+            if (null !== $selectedSegment) {
+                $title = $selectedSegment->getName();
+            } else {
+                $title = $this->translator->trans('Select a segment');
```

Comment:
This is a weird translation string. Also, use the `|trans` twig filter for the title as well instead of translating here.